src/shapes/textbox.class.js

```diff
@@ -78,6 +86,28 @@
      */
     splitByGrapheme: false,
 
+    /**
+     * While editing handle differently
```

Comment:
this comment makes little sense.
If it means `this logic doesn't apply while editing because ...` let's write that

Reply:
done